src/index.html

```diff
@@ -10,13 +10,19 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Stop watch</title>
+
+    <title>Stopwatch</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The href attribute points to an SCSS file. Typically, browsers cannot directly interpret SCSS files, so you should ensure that this SCSS file is compiled into a CSS file and link to the resulting CSS file instead.